tests/ignite/test_utils.py

```diff
@@ -120,8 +120,8 @@ def test_setup_logger(capsys, dirname):
 
     def _test(stream):
 
-        trainer.logger = setup_logger("trainer", stream=stream, filepath=fp)
-        evaluator.logger = setup_logger("evaluator", stream=stream, filepath=fp)
+        trainer.logger = setup_logger("trainer", stream=stream, filepath=fp, reset=True)
```

Comment:
Can we add a test as in the issue: use auto method by default than override the logger and ensure that nothing in stdout ?